Assets/Placenote/Examples/ARKitPlaneSaver/PlacenoteARGeneratePlane.cs

```diff
@@ -20,10 +20,22 @@ public class ARPlaneMesh
         public string id;
     }
 
+	[System.Serializable]
```

Comment:
move this into the spatial capture folder

Reply:
Seperate namespaces created. One for the examples and the other for editor scripts